packages/documentation-site/patternfly-docs/content/PatternFly-AI/patternfly-mcp.md

```diff
@@ -0,0 +1,95 @@
+
+MCP stands for Model Context Protocol, an open standard that allows AI assistants to securely and accurately access external data sources and tools. The PatternFly MCP is a server that connects your AI assistant (like those within Cursor or Claude Code) directly to official PatternFly documentation. Instead of relying on a model's potentially outdated or overgeneralized knowledge, your assistant can query the PatternFly MCP to get answers based on real, up-to-date, and verified sources.
+
+## What are the benefits of the PatternFly MCP?
+
+This tool is valuable for developers, designers, and other stakeholders, like product managers. This can look different depending on your needs, but the primary benefit is trust. The PatternFly MCP helps reduce AI "hallucinations" by ensuring the answers you receive are quoted directly from our documentation and are linked to the correct sources.
+
+- **For developers:** Receive quick and accurate information about component props, implementation details, and accessibility requirements. The PatternFly MCP can also help validate component usage or choose the right component for a specific need.
+- **For designers:** Quickly reference design guidelines and usage patterns while you work. You can ask for best practices, such as "Which component should I use for a gallery view?", and get answers rooted in our official standards.
+- **For stakeholders:** By embedding the design system directly into the development workflow, the PatternFly MCP ensures consistency and adherence to design and accessibility standards across all projects. This speeds up development and review cycles by reducing errors and guesswork.
+
+## What does the PatternFly MCP provide? 
+There are 2 main tools provided by the PatternFly MCP:
+- `usePatternFlyDocs`: Used to answer high-level questions about components, design guidelines, and accessibility. 
+- `fetchDocs`: Used to retrieve the full documentation for a specific component via a URL provided by `usePatternFlyDocs`.
+
+## Get started
+You can install the PatternFly MCP in your preferred development environment. Here are the instructions for popular clients.
+
+### Cursor
+1. While in Cursor, press **cmd** + **shift** +**P**. 
+1. Search for and select **View: Open MCP Settings**. 
+1. At the bottom of the settings menu, click **New MCP Server** to open the `mcp.json` file.
+1. Add the following JSON block within the `mcpServers` object. If `mcpServers` already exists, just add the `patternfly-docs` block inside it.
+
+    ``` {
+    "mcpServers": {
+        "patternfly-docs": {
+        "command": "npx",
+        "args": [
+            "-y",
+            "@patternfly/patternfly-mcp"
+        ],
+        "description": "PatternFly React development rules and documentation"
+        }
+    }
+    }
+    ```
```

Comment:
It looks like the very opening { got cut off and the indentation got a little thrown off following that:
```suggestion
    ``` 
    {
      "mcpServers": {
        "patternfly-docs": {
          "command": "npx",
          "args": [
            "-y",
            "@patternfly/patternfly-mcp"
          ],
          "description": "PatternFly React development rules and documentation"
        }
      }
    }
    ```
```